Mathlib/Analysis/Calculus/FDeriv/Add.lean

```diff
@@ -624,7 +670,7 @@ theorem DifferentiableAt.sub_const (hf : DifferentiableAt 𝕜 f x) (c : F) :
   (hf.hasFDerivAt.sub_const c).differentiableAt
 #align differentiable_at.sub_const DifferentiableAt.sub_const
 
-@[simp]
+@[deprecated DifferentiableAt.sub_iff_left]
```

Comment:
Please add `(since := ... )` clauses to your deprecation. Note that we have a `deprecated alias` code snippet to help you with this (although currently it doesn't do exactly what you want here)
```suggestion
@[deprecated DifferentiableAt.sub_iff_left (since := 2024-07-11)]
```

Also please golf the proof of the lemmas you deprecate so that people know how to replace the deprecated lemmas by the non-deprecated ones